invenio_cli/helpers/commands.py

```diff
@@ -195,22 +212,27 @@ def services(self, force):
         ]
         subprocess.run(command, check=True)
 
-        # If re-run, this throws an error
+        # Without the self.cli_config.get_services_setup() check
```

Comment:
Moderate: This comment is repeated for every command. I would just put it once on the top and explain it applies to all. Or is it like that so as they become idempotent we can take out one by one? If so I agree with it, but we should open a linked issue.

Reply:
The latter was what I had in mind. I'll add the issue number for each, so it's clearer.